modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -0,0 +1,94 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-configuring-distr-tracing-tempo_{context}"]
+= Configuring the distributed tracing platform (Tempo)
+
+You can expose tracing data to the {TempoName} stack by appending a named element and `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+
+The following example configures the `ServiceMeshControlPlane` resource to define an extensionProvider `zipkin` for sending tracing spans to a TempoStack. You can create a TempoStack instance in the `tracing-system` namespace _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+
+.Prerequisites
+
+You have installed the {TempoOperator} and {SMProductName} Operator in the `openshift-operators` namespace and have created namespaces such as `istio-system` and `tracing-system`.
+
+.Procedure
+
+. Configure the `ServiceMeshControlPlane` resource to define an extension provider:
++
+[source,yaml]
+----
+kind: ServiceMeshControlPlane
+apiVersion: maistra.io/v2
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+...
```

Comment:
```suggestion
# ...
```

Reply:
Nice job using the ellipses here. Per the [contributing](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting) guidance, comment out ellipses when truncating YAML to avoid creating a document end marker in the YAML.

Reply:
Is there a place to end the ellipses?